SoftLayer/CLI/server/detail.py

```diff
@@ -80,7 +80,7 @@ def cli(env, identifier, passwords, price):
 
     if price:
         table.add_row(['price rate',
-                       result['billingItem']['recurringFee']])
+                       result['billingItem']['nextInvoiceTotalRecurringAmount']])
```

Comment:
flake8 is complaining about this line since it's over 80 characters long.

I recommend doing this:
```python
        table.add_row(['price rate',
                       utils.lookup(result,
                                    'billingItem',
                                    'nextInvoiceTotalRecurringAmount')])
```